dom_wasm.go

```diff
@@ -1088,14 +1088,16 @@ func (w *window) Stop() {
 
 // TODO reuse util.EventTarget
 
-func (w *window) AddEventListener(typ string, useCapture bool, listener func(Event)) func(o js.Value) {
-	wrapper := func(o js.Value) { listener(wrapEvent(o)) }
-	w.Call("addEventListener", typ, wrapper, useCapture)
-	return wrapper
+func (w *window) AddEventListener(typ string, useCapture bool, listener func(Event)) js.Callback {
+	wrapper := func(o []js.Value) { listener(wrapEvent(o[0])) }
+	cb := js.NewCallback(wrapper)
```

Comment:
I think these should be NewEventCallback() and accept the callback flags too.

Reply:
Makes sense, will add `js.EventCallbackFlag` as an argument.

Reply:
Just curious, what value does adding the `js.EventCallbackFlag` when the dom.Event object already has those methods available?

For example: `event.PreventDefault()`

This seems to introduce two ways to do the same thing.